libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/ObjectDetectionModelOverview.tsx

```diff
@@ -44,6 +50,8 @@ export function getSelectableClassNames(dataset: IDataset): IComboBoxOption[] {
 export interface IObjectDetectionWidgetsProps {
   classNames: IProcessedStyleSet<IModelOverviewStyles>;
   dataset: IDataset;
+  modelOverview: any; // avoided ModelOverview type due to circular imports
```

Comment:
instead of passing parent class like this, it's better to pass the individual properties/variables from that class directly and functions to update the parent class